deploy/services/cli/README.md

```diff
@@ -0,0 +1,283 @@
+# DTaaS Services CLI
+
+A command-line tool for managing DTaaS platform services including MongoDB,
+InfluxDB, RabbitMQ, and Grafana.
+
+## Features
+
+* **Project Initialization:** Generate project structure with config and data directories
+* **Automated Setup:** One command setup of TLS certificates and permissions
+* **Service Management:** Start, stop, and check status of all services
+* **User Management:** Easy creation of user accounts in InfluxDB and RabbitMQ
+* **Cross platform:** Works on Linux, macOS, and Windows
+* **Configuration-driven:** Reads settings from `config/services.env`
+
+## Installation
+
+### Prerequisites
+
+* Python 3.10 or higher
+* Docker and Docker Compose
+* TLS certificates
+
+### Install from Wheel Package
+
+Install the standalone wheel package using pip:
+
+```bash
+pip install dtaas_services-0.1.0-py3-none-any.whl
+```
+
+This installs the `dtaas-services` command.
+
+To verify the installation:
+
+```bash
+dtaas-services --help
+```
+
+## Quick Start
+
+1. Navigate to where you want to set up the services and generate the project
+structure and run:
+
+   ```bash
+   dtaas-services generate-project 
+   ```
+
+   This creates:
+   * `config/` directory with configuration templates
+   * `data/` directory for service data
+   * `compose.services.secure.yml` for Docker Compose
+
+2. Update `config/services.env` with your environment values:
+   * `SERVICES_UID` - User ID for service file ownership
+   * `SERVICES_GID` - Group ID for service file ownership
+   * `SERVER_DNS` - Your server hostname
+   * Port numbers for each service
+
+3. Update `config/credentials.csv` with user accounts (format: `username,password`)
+
+## Usage
+
+### Service Setup
+
+After generating the project and configuring your settings:
+
+```bash
+dtaas-services setup
```

Comment:
```bash
(.venv-for-dtaas-services) ➜  test git:(pr-1377) ✗ dtaas-services setup 
This script must be run as root (Linux/MacOS). Try: sudo -E env PATH="$PATH" dtaas-services setup
(.venv-for-dtaas-services) ➜  test git:(pr-1377) ✗ sudo dtaas-services setup
sudo: dtaas-services: command not found
```

This shouldn't be a requirement. The default should be to run it as current user. If user does not have access to docker daemon, they can run using `sudo dtaas-services setup`.